### PR TITLE
Fixes and Game Index

### LINKED ISSUE #11: Add nice metadata
- [x] Add metata by following (this tutorial](https://www.lewagon.com/blog/setup-meta-tags-rails)
- [x] Add custom title and description for each page
- [x] Find a nice logo

### LINKED ISSUE #18: Remove active admin comments

### PR DESCRIPTION
**Pull request by benjyarel**
Hi  Germain 🕵🏽‍♂️  !

DB : 

- Added Franchise and Genre to Games table
- Change Year as a string into Games tables ( Maxime wants to write "TBA" " 2 Octobre 2019")

Active Admin : 

- Update New/edit forms for new columns in the DB.
-Delete comments sections and re-order the Navbar.
-Added Buttons linked to the root in the dashboard, to the index page of each model.

Meta-Data : 
- Fav-icon is in place
- Makes things you said about meta_title and Meta-product **_TO-DO : Ask Maxime the description and the picture he wants for OG Meta_data._**

Games Index : 
-I did the game-card and the "bootstrap layout" . I tried to be a maximum like Maxime's Figma.
**_- To-DO : Responsive behavior of course 💪  _**

close #18 
close #11

### CODE REVIEW

**Comment by benjyarel** [comment]
I Worked this morning on the mobile version of Game Index : 
close #8

File: app/admin/movies.rb [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ActiveAdmin.register Movie do
+  menu priority: 1
   config.filters = false
   config.per_page = 15
 
+  action_item :view_site do
+    link_to "Go to movies page", "/movies", target: "_blank"
```
Comment (grmnlrt) [inline app/admin/movies.rb]: 👍

File: app/admin/games.rb [hunk 2]
```diff
@@ -1,5 +1,11 @@
 ActiveAdmin.register Game do
-  config.filters = false # disabled filters for games
+  menu priority: 2
+  config.filters = false
+  config.per_page = 15
+
+  action_item :view_site do
+    link_to "Go to games page", "/games", target: "_blank"
```
Comment (grmnlrt) [inline app/admin/games.rb]: 👍

File: app/assets/stylesheets/components/_game-card.scss [hunk 3]
```diff
@@ -0,0 +1,98 @@
+.game-card{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space between `.game-card` and `{`

File: app/assets/stylesheets/components/_game-card.scss [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  @media (max-width: 768px) {
+    display: none,
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: `,` instead of `;`

File: app/assets/stylesheets/components/_game-card.scss [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+    margin : auto;
+    margin-bottom: 3rem;
+    max-width: 800px;
+    max-height: 220px;
+    display: flex;
+
+
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: No needs to double line break

File: app/assets/stylesheets/components/_game-card.scss [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  .picture{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+    height: 200px;
+    width : 315px;
+  }
+
+  img{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+    object-fit: cover;
+    border-radius: 3px;
+  }
+
+}
+
+.game-content, .game-content-mobile{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+  margin-left: 1rem;
+  font-size: $body-font;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: Do we need this ? `$body-font` is the default font for all text

File: app/assets/stylesheets/components/_game-card.scss [hunk 10]
```diff
@@ -0,0 +1,98 @@
+.game-card{
+  @media (max-width: 768px) {
+    display: none,
+  }
+
+    margin : auto;
+    margin-bottom: 3rem;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: Maybe we should use the [8 point rule](https://builttoadapt.io/intro-to-the-8-point-grid-system-d2573cde8632) for consistency

File: app/assets/stylesheets/components/_game-card.scss [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+
+  img{
+    object-fit: cover;
+    border-radius: 3px;
+  }
+
+}
+
+.game-content, .game-content-mobile{
+  margin-left: 1rem;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: 8 point rule

File: app/assets/stylesheets/components/_game-card.scss [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+  font-size: $body-font;
+  font-size: 16px;
+  font-weight: 400;
+
+  h4{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+    font-size: 20px;
+    font-family: $headers-font;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: Same, I think we don't need this

File: app/assets/stylesheets/components/_game-card.scss [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+    margin-bottom: 1,5rem;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: 8 point rule

File: app/assets/stylesheets/components/_game-card.scss [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  p{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+    margin: 0;
+  }
+  span{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+      color: gray;
+      font-weight: 300;
+    }
+
+}
+
+.game-published{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: remove break line

File: app/assets/stylesheets/components/_game-card.scss [hunk 19]
```diff
@@ -0,0 +1,98 @@
...
+  padding: 1rem 0px;
+
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: remove break line

File: app/assets/stylesheets/components/_game-card.scss [hunk 20]
```diff
@@ -0,0 +1,98 @@
...
+  font-size : 12px ;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: no space between `12px` and `;`

File: app/assets/stylesheets/components/_game-card.scss [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.game-card-mobile{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+  background: $lightest-gray;
+
+  @media (min-width: 768px){
+    display: none;
+  }
+
+  }
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: indentation

File: app/assets/stylesheets/components/_game-card.scss [hunk 23]
```diff
@@ -0,0 +1,98 @@
...
+  .game-content-mobile{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 24]
```diff
@@ -0,0 +1,98 @@
...
+
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: no break line

File: app/assets/stylesheets/components/_game-card.scss [hunk 25]
```diff
@@ -0,0 +1,98 @@
...
+    margin-top: 20px;
+    margin-bottom: 80px;
+  }
+
+  img{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 26]
```diff
@@ -0,0 +1,98 @@
...
+    width : 95%;
+    height: 350px;
+    margin: 15px;
+    object-fit: cover;
+    border-radius: 3px;
+  }
+  .picture-mobile{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space

File: app/assets/stylesheets/components/_game-card.scss [hunk 27]
```diff
@@ -0,0 +1,98 @@
...
+    height: 350px;
+  }
+
+  .title-mobile{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: same

File: app/assets/stylesheets/components/_game-card.scss [hunk 28]
```diff
@@ -0,0 +1,98 @@
...
+    padding: 10px 0px;
+  }
+
+h4{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: missing space
And be careful ⚠️: Here you're setting style for each `h4` on the website.

File: app/assets/stylesheets/components/_game-card.scss [hunk 29]
```diff
@@ -0,0 +1,98 @@
...
+  color: #f2f2f2;
+  text-shadow: 0px 1px 5px #a7a7a7;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: Don't understand this. Why do you want to add text shadow?

File: app/assets/stylesheets/components/_game-card.scss [hunk 30]
```diff
@@ -0,0 +1,98 @@
...
+  position: relative;
+  left: 25px;
+  bottom: 50px;
+}
+
+  @media (max-width:450px){
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: indentation and space

File: app/assets/stylesheets/components/_game-card.scss [hunk 31]
```diff
@@ -0,0 +1,98 @@
...
+    img{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: space

File: app/assets/stylesheets/components/_game-card.scss [hunk 32]
```diff
@@ -0,0 +1,98 @@
...
+      max-height: 300px;
+    }
+    .picture-mobile{
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: space

File: app/views/games/index.html.erb [hunk 33]
```diff
@@ -1,5 +1,58 @@
...
+              <p><span>Genre </span><%= game.genre %></p>
+            <% end %>
+            <% if game.year? %>
+              <p class="game-published"><span>date de parution </span><%= game.year%></p>
+            <% end %>
+          </div>
+        </div>
+        <div class="game-card-mobile">
+          <div class="picture-mobile">
+            <%= instance_image_tag game.photo, effect:"gradient_fade,y_-0.5,b_rgb:050505", crop: :fill%>
```
Comment (grmnlrt) [inline app/views/games/index.html.erb]: what `effect` does?

**Comment by grmnlrt** [comment]
I just talk to Max, he told me he doesn't need `Franchise` column for games

File: app/assets/stylesheets/components/_game-card.scss (hunk 29 above)
Comment (benjyarel) [reply app/assets/stylesheets/components/_game-card.scss]: Capture d’écran 2019-12-30 à 15 08 31
Capture d’écran 2019-12-30 à 15 09 21

I thought, in the mobile version, it was more "soft" to have lightening under the white text, I gave you screenshots with or without, tell me :)

File: app/views/games/index.html.erb (hunk 33 above)
Comment (benjyarel) [reply app/views/games/index.html.erb]: As you can see in the previous comment, I added a gradient filter over the image, to see better the white title. That's how Cloudinary handles special effects on images.

File: app/assets/stylesheets/components/_game-card.scss (hunk 11 above)
Comment (benjyarel) [reply app/assets/stylesheets/components/_game-card.scss]: I changed to 16 px :) but isn't the same thing ?

File: app/assets/stylesheets/components/_game-card.scss (hunk 28 above)
Comment (benjyarel) [reply app/assets/stylesheets/components/_game-card.scss]: Yep, sorry for the mistake, I created a class .title-inside-mobile, for better scopre :-/

**Comment by benjyarel** [comment]
> I just talk to Max, he told me he doesn't need `Franchise` column for games

remove-column :franchise done.
I corrected what you said, I'm just waiting you feedback about text-shadow !

File: app/assets/stylesheets/components/_game-card.scss (hunk 29 above)
Comment (grmnlrt) [reply app/assets/stylesheets/components/_game-card.scss]: mmmh OK I see. Not a bad idea, but for the moment I see the text with shadow on desktop version, like on the screenshot above

File: app/assets/stylesheets/components/_game-card.scss [hunk 34]
```diff
@@ -0,0 +1,100 @@
+.game-card {
+  @media (max-width: 768px) {
+    display: none;
+  }
+
+    margin : auto;
+    margin-bottom: 24px;
+    max-width: 800px;
+    max-height: 220px;
+    display: flex;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: indentation

File: app/assets/stylesheets/components/_game-card.scss [hunk 35]
```diff
@@ -0,0 +1,100 @@
...
+  @media (min-width: 768px) {
+    display: none;
+  }
+
+  img {
+    width : 100%;
+    height: 350px;
+    margin-bottom: 16px;
+    object-fit: cover;
+    border-radius: 3px;
```
Comment (grmnlrt) [inline app/assets/stylesheets/components/_game-card.scss]: We don't need `border-radius` here

File: app/assets/stylesheets/components/_game-card.scss (hunk 29 above)
Comment (grmnlrt) [reply app/assets/stylesheets/components/_game-card.scss]: Ok looks way better now, thanks for your work 🙏
Keep it like this for the moment, we'll ask to Max which version he prefer

**Comment by grmnlrt** [comment]
Oh and one last thing, we need to translate all text in english (saw that with Max this afternoon).

**Pull request merged**